src/stores/ProfileFittingStore.ts

```diff
@@ -0,0 +1,522 @@
+import {action, computed, observable, makeObservable} from "mobx";
+import {FittingFunction, FittingContinuum} from "components/SpectralProfiler/ProfileFittingComponent/ProfileFittingComponent";
+import {SpectralProfileWidgetStore} from "stores/widgets";
+import {Point2D} from "models";
+import * as GSL from "gsl_wrapper";
```

Comment:
the GSL import should go after the mobx import, and before any of our internal imports